__docs__/wonder-blocks-labeled-field/labeled-field.stories.tsx

```diff
@@ -64,6 +64,7 @@ export const Default: StoryComponentType = {
         field: <TextField value="" onChange={() => {}} />,
         label: "Name",
         description: "Helpful description text.",
+        contextLabel: "Context label",
```

Comment:
Should there be a `contextLabel` story so users don't have to hunt around for it? Maybe it could document the `optional` use case?

Reply:
Great idea! Added a story and docs for the Context Label :) `?path=/docs/packages-labeledfield--docs&globals=theme:thunderblocks#context-label`